### ISSUE TITLE
Getting an error while fastlane setup

### ISSUE THREAD

**Issue by Quarezz** [fastlane/fastlane#1254]
Hi, 
I've had fastlane already setup, but deleted all folders and files and decided to do the setup again.

After step with choosing team for my Apple ID - I get error:
```
Multiple teams found, please enter the number of the team you want to use: 
1) *** "******" (Company/Organization)
2) *** "****" (Company/Organization)
1
[15:43:30]: Error occurred with the setup program! Reverting changes now!
[15:43:30]: Deleting the 'fastlane' folder
Successfully submitted a crash report. If this is a problem with one of the tools specifically,
please submit an issue on GitHub and attach the following number to it: '953b655e64d1bd92d643a956c09baba8'
The crash report has been stored locally '/tmp/sentry_953b655e64d1bd92d643a956c09baba8.json'


/Users/quarezz/.rvm/rubies/ruby-2.2.1/lib/ruby/gems/2.2.0/gems/spaceship-0.19.3/lib/spaceship/client.rb:268:in `parse_response': [!] {"data"=>nil, "messages"=>{"warn"=>nil, "error"=>["Unauthorized access"], "info"=>nil}, "statusCode"=>"ERROR"} (Spaceship::Client::UnexpectedResponse)
	from /Users/quarezz/.rvm/rubies/ruby-2.2.1/lib/ruby/gems/2.2.0/gems/spaceship-0.19.3/lib/spaceship/tunes/tunes_client.rb:45:in `teams'
	from /Users/quarezz/.rvm/rubies/ruby-2.2.1/lib/ruby/gems/2.2.0/gems/spaceship-0.19.3/lib/spaceship/tunes/tunes_client.rb:83:in `select_team'
	from /Users/quarezz/.rvm/rubies/ruby-2.2.1/lib/ruby/gems/2.2.0/gems/spaceship-0.19.3/lib/spaceship/tunes/spaceship.rb:30:in `select_team'
	from /Users/quarezz/.rvm/rubies/ruby-2.2.1/lib/ruby/gems/2.2.0/gems/fastlane-1.55.0/lib/fastlane/setup/setup_ios.rb:173:in `detect_if_app_is_available'
	from /Users/quarezz/.rvm/rubies/ruby-2.2.1/lib/ruby/gems/2.2.0/gems/fastlane-1.55.0/lib/fastlane/setup/setup_ios.rb:29:in `run'
	from /Users/quarezz/.rvm/rubies/ruby-2.2.1/lib/ruby/gems/2.2.0/gems/fastlane-1.55.0/lib/fastlane/setup/setup.rb:19:in `run'
	from /Users/quarezz/.rvm/rubies/ruby-2.2.1/lib/ruby/gems/2.2.0/gems/fastlane-1.55.0/bin/fastlane:41:in `block (2 levels) in run'
	from /Users/quarezz/.rvm/rubies/ruby-2.2.1/lib/ruby/gems/2.2.0/gems/commander-4.3.5/lib/commander/command.rb:178:in `call'
	from /Users/quarezz/.rvm/rubies/ruby-2.2.1/lib/ruby/gems/2.2.0/gems/commander-4.3.5/lib/commander/command.rb:178:in `call'
	from /Users/quarezz/.rvm/rubies/ruby-2.2.1/lib/ruby/gems/2.2.0/gems/commander-4.3.5/lib/commander/command.rb:153:in `run'
	from /Users/quarezz/.rvm/rubies/ruby-2.2.1/lib/ruby/gems/2.2.0/gems/commander-4.3.5/lib/commander/runner.rb:428:in `run_active_command'
	from /Users/quarezz/.rvm/rubies/ruby-2.2.1/lib/ruby/gems/2.2.0/gems/fastlane_core-0.36.1/lib/fastlane_core/ui/fastlane_runner.rb:23:in `run!'
	from /Users/quarezz/.rvm/rubies/ruby-2.2.1/lib/ruby/gems/2.2.0/gems/commander-4.3.5/lib/commander/delegates.rb:15:in `run!'
	from /Users/quarezz/.rvm/rubies/ruby-2.2.1/lib/ruby/gems/2.2.0/gems/fastlane-1.55.0/bin/fastlane:150:in `run'
	from /Users/quarezz/.rvm/rubies/ruby-2.2.1/lib/ruby/gems/2.2.0/gems/fastlane-1.55.0/bin/fastlane:156:in `<top (required)>'
	from /Users/quarezz/.rvm/rubies/ruby-2.2.1/bin/fastlane:23:in `load'
	from /Users/quarezz/.rvm/rubies/ruby-2.2.1/bin/fastlane:23:in `<main>'
	from /Users/quarezz/.rvm/gems/ruby-2.2.1/bin/ruby_executable_hooks:15:in `eval'
	from /Users/quarezz/.rvm/gems/ruby-2.2.1/bin/ruby_executable_hooks:15:in `<main>'
```

**Comment by kevboh** [comment]
Also getting this. The Apple ID I'm using does not have multiple teams. Same stack trace.

**Comment by Quarezz** [comment]
Yeah, tried on several test projects - no luck. Maybe Apple changed something and spaceship doesn't work on setup right now...

**Comment by iltercengiz** [comment]
I'm getting an error during setup too but mine is different

```
/Users/Ilter/.rvm/rubies/ruby-2.2.1/lib/ruby/2.2.0/net/http.rb:923:in `connect': [!] SSL_connect returned=1 errno=0 state=SSLv3 read server certificate B: certificate verify failed (Faraday::SSLError)
	from /Users/Ilter/.rvm/rubies/ruby-2.2.1/lib/ruby/2.2.0/net/http.rb:923:in `block in connect'
	from /Users/Ilter/.rvm/rubies/ruby-2.2.1/lib/ruby/2.2.0/timeout.rb:74:in `timeout'
	from /Users/Ilter/.rvm/rubies/ruby-2.2.1/lib/ruby/2.2.0/net/http.rb:923:in `connect'
	from /Users/Ilter/.rvm/rubies/ruby-2.2.1/lib/ruby/2.2.0/net/http.rb:863:in `do_start'
	from /Users/Ilter/.rvm/rubies/ruby-2.2.1/lib/ruby/2.2.0/net/http.rb:852:in `start'
	from /Users/Ilter/.rvm/rubies/ruby-2.2.1/lib/ruby/2.2.0/net/http.rb:1375:in `request'
	from /Users/Ilter/.rvm/rubies/ruby-2.2.1/lib/ruby/2.2.0/net/http.rb:1133:in `get'
	from /Users/Ilter/.rvm/gems/ruby-2.2.1/gems/faraday-0.9.2/lib/faraday/adapter/net_http.rb:80:in `perform_request'
	from /Users/Ilter/.rvm/gems/ruby-2.2.1/gems/faraday-0.9.2/lib/faraday/adapter/net_http.rb:40:in `block in call'
	from /Users/Ilter/.rvm/gems/ruby-2.2.1/gems/faraday-0.9.2/lib/faraday/adapter/net_http.rb:87:in `with_net_http_connection'
	from /Users/Ilter/.rvm/gems/ruby-2.2.1/gems/faraday-0.9.2/lib/faraday/adapter/net_http.rb:32:in `call'
	from /Users/Ilter/.rvm/gems/ruby-2.2.1/gems/faraday-cookie_jar-0.0.6/lib/faraday/cookie_jar.rb:22:in `call'
	from /Users/Ilter/.rvm/gems/ruby-2.2.1/gems/faraday_middleware-0.10.0/lib/faraday_middleware/response_middleware.rb:30:in `call'
	from /Users/Ilter/.rvm/gems/ruby-2.2.1/gems/faraday_middleware-0.10.0/lib/faraday_middleware/response_middleware.rb:30:in `call'
	from /Users/Ilter/.rvm/gems/ruby-2.2.1/gems/faraday_middleware-0.10.0/lib/faraday_middleware/response_middleware.rb:30:in `call'
	from /Users/Ilter/.rvm/gems/ruby-2.2.1/gems/faraday-0.9.2/lib/faraday/rack_builder.rb:139:in `build_response'
	from /Users/Ilter/.rvm/gems/ruby-2.2.1/gems/faraday-0.9.2/lib/faraday/connection.rb:377:in `run_request'
	from /Users/Ilter/.rvm/gems/ruby-2.2.1/gems/faraday-0.9.2/lib/faraday/connection.rb:140:in `get'
	from /Users/Ilter/.rvm/gems/ruby-2.2.1/gems/spaceship-0.19.3/lib/spaceship/portal/portal_client.rb:22:in `api_key'
	from /Users/Ilter/.rvm/gems/ruby-2.2.1/gems/spaceship-0.19.3/lib/spaceship/portal/portal_client.rb:37:in `send_login_request'
	from /Users/Ilter/.rvm/gems/ruby-2.2.1/gems/spaceship-0.19.3/lib/spaceship/client.rb:173:in `login'
	from /Users/Ilter/.rvm/gems/ruby-2.2.1/gems/spaceship-0.19.3/lib/spaceship/client.rb:57:in `login'
	from /Users/Ilter/.rvm/gems/ruby-2.2.1/gems/spaceship-0.19.3/lib/spaceship/portal/spaceship.rb:22:in `login'
	from /Users/Ilter/.rvm/gems/ruby-2.2.1/gems/spaceship-0.19.3/lib/spaceship/portal/spaceship.rb:77:in `login'
	from /Users/Ilter/.rvm/gems/ruby-2.2.1/gems/fastlane-1.57.0/lib/fastlane/setup/setup_ios.rb:168:in `detect_if_app_is_available'
	from /Users/Ilter/.rvm/gems/ruby-2.2.1/gems/fastlane-1.57.0/lib/fastlane/setup/setup_ios.rb:29:in `run'
	from /Users/Ilter/.rvm/gems/ruby-2.2.1/gems/fastlane-1.57.0/lib/fastlane/setup/setup.rb:19:in `run'
	from /Users/Ilter/.rvm/gems/ruby-2.2.1/gems/fastlane-1.57.0/bin/fastlane:51:in `block (2 levels) in run'
	from /Users/Ilter/.rvm/gems/ruby-2.2.1/gems/commander-4.3.5/lib/commander/command.rb:178:in `call'
	from /Users/Ilter/.rvm/gems/ruby-2.2.1/gems/commander-4.3.5/lib/commander/command.rb:178:in `call'
	from /Users/Ilter/.rvm/gems/ruby-2.2.1/gems/commander-4.3.5/lib/commander/command.rb:153:in `run'
	from /Users/Ilter/.rvm/gems/ruby-2.2.1/gems/commander-4.3.5/lib/commander/runner.rb:428:in `run_active_command'
	from /Users/Ilter/.rvm/gems/ruby-2.2.1/gems/fastlane_core-0.36.1/lib/fastlane_core/ui/fastlane_runner.rb:23:in `run!'
	from /Users/Ilter/.rvm/gems/ruby-2.2.1/gems/commander-4.3.5/lib/commander/delegates.rb:15:in `run!'
	from /Users/Ilter/.rvm/gems/ruby-2.2.1/gems/fastlane-1.57.0/bin/fastlane:150:in `run'
	from /Users/Ilter/.rvm/gems/ruby-2.2.1/gems/fastlane-1.57.0/bin/fastlane:156:in `<top (required)>'
	from /Users/Ilter/.rvm/gems/ruby-2.2.1/bin/fastlane:23:in `load'
	from /Users/Ilter/.rvm/gems/ruby-2.2.1/bin/fastlane:23:in `<main>'
	from /Users/Ilter/.rvm/gems/ruby-2.2.1/bin/ruby_executable_hooks:15:in `eval'
	from /Users/Ilter/.rvm/gems/ruby-2.2.1/bin/ruby_executable_hooks:15:in `<main>'
```

This occurs after I enter my Apple ID and password. Any info about that?

**Comment by KrauseFx** [comment]
These are all different errors. @iltercengiz your issue is covered here https://github.com/fastlane/fastlane/blob/master/docs/FAQs.md#im-getting-an-ssl-error

**Comment by iltercengiz** [comment]
@KrauseFx tried it but no chance. Still getting the same error. Also I've updated all gems.

**Comment by iltercengiz** [comment]
Fixed the SSL error. A little research led me here:
https://github.com/rvm/rvm/issues/3330 and then here:
https://toadle.me/2015/04/16/fixing-failing-ssl-verification-with-rvm.html

TL;DR: If ruby is managed by rvm and the installed version is the binary version, ruby is searching the certs in the wrong directory. Instead, install ruby from source locally by doing
`rvm install 2.2.1 --disable-binary`
This way it will find certs.

**Comment by vpolouchkine** [comment]
It looks like there hasn’t been any recent activity on this issue in some time, so I’m closing it. If this is still a problem, feel free to re-open a new issue and I’d be happy to help :+1:
